src/main/java/uk/gov/companieshouse/officerfiling/api/validation/OfficerAppointmentValidator.java

```diff
@@ -89,11 +89,17 @@ public void validateRequiredDtoFields(HttpServletRequest request, List<ApiError>
         validateNationality2(request, errorList, dto);
         validateNationality3(request, errorList, dto);
         validateNationalityLength(request, errorList, dto);
-        validateRequiredResidentialAddressFields(request, errorList, dto);
-        validateCorrespondenceAddressFields(request, errorList, dto);
         validateAppointmentDate(request, errorList, dto);
         validateProtectedDetails(request, errorList, dto);
         validateConsentToAct(request, errorList, dto);
+
+        if (Boolean.FALSE.equals(dto.getIsHomeAddressSameAsServiceAddress())) {
```

Comment:
Should this not validate if the flag is undefined? Defaulting to false. So !Boolean.True.equals()

Reply:
You're correct Andre - I'd previously had a null check as well and had removed it where I shouldn't have, but your suggestion is cleaner anyway. Have made that change and re-tested and the null case works as it should now. Nice catch!